calcfetch

```diff
@@ -1,8 +1,8 @@
-#!/bin/bash
+#!/usr/bin/env bash
```

Comment:
Bash isn't required to be in a specific dir, however /usr/bin/env is, so this will work in more places.